fluent-kafka-streams-tests-junit5/src/test/java/com/bakdata/fluent_kafka_streams_tests/junit5/test_applications/WordCount.java

```diff
@@ -0,0 +1,57 @@
+package com.bakdata.fluent_kafka_streams_tests.junit5.test_applications;
+
+import java.util.Arrays;
+import java.util.Properties;
+import java.util.regex.Pattern;
+import lombok.Getter;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.streams.Topology;
+import org.apache.kafka.streams.kstream.KStream;
+import org.apache.kafka.streams.kstream.KTable;
+import org.apache.kafka.streams.kstream.Produced;
+
+public class WordCount {
```

Comment:
i think this could be in some base test (similar to the individual test classes) and then included in all three version, instead of having three duplicates.

Reply:
I think the effort is not worth the benefit (if there is any).